metal3-dev/run.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -xe
 
+source $(dirname $(dirname $0))/common.sh
```

Comment:
This doesn't work if you cd into metal3-dev then run the script, I think you need something similar to SCRIPTDIR in common.sh e.g `source $(dirname $(dirname $0))/common.sh`